### ISSUE TITLE
Style for terra-clinical-item-view making the text grow larger in smaller viewports

### ISSUE THREAD

**Issue by kschuste** [cerner/terra-clinical#414]
## Description
There is a styling for the terra-clinical-item-view that we are questioning on if it should be the way it is or if it should be updated.
https://github.com/cerner/terra-clinical/blob/441aa4b3fbec4c1fe1efa82b215b7ae97deaa88f/packages/terra-clinical-item-view/src/_mixins.scss#L1

With that style, it flexes the size of the text you are showing based off the width of its window.  It is used in the terra-clinical-item-view here for the rows after the first row.
https://github.com/cerner/terra-clinical/blob/441aa4b3fbec4c1fe1efa82b215b7ae97deaa88f/packages/terra-clinical-item-view/src/ItemView.jsx#L89
https://github.com/cerner/terra-clinical/blob/441aa4b3fbec4c1fe1efa82b215b7ae97deaa88f/packages/terra-clinical-item-view/src/ItemView.scss#L113

The first item for the ItemView uses the terra-clinical-tiny-heading, which does not have this flexing of the text size.
https://github.com/cerner/terra-clinical/blob/441aa4b3fbec4c1fe1efa82b215b7ae97deaa88f/packages/terra-clinical-item-view/src/_mixins.scss#L10

This is causing some confusion that the secondary rows are gaining more emphasis over the first row.  Should the text be staying the same height, should the first row get its size to grow, or is it correct?

## Additional Context / Screenshots
<img width="565" alt="screen shot 2018-10-31 at 3 46 20 pm" src="https://user-images.githubusercontent.com/1711642/47817640-2ab70a00-dd24-11e8-8bc0-88fb3df0d577.png">

## Expected Behavior
We would expect all the text sizes for all the rows to remain the same whether they all resize or don't resize, or that the first row would get the text height flexing and not the secondary rows.

**Comment by mjhenkes** [comment]
First off you should not be re-using the _mixins.scss file. It's intended for internal use only. What are you trying to accomplish that the base terra-clinical-item-view doesn't support?

**Comment by kschuste** [comment]
We are wanting to show more inline on our second row than just text such as an icon after the text and not in-front of it.  By using the ItemView.Display, it only provides the icon in the front.  Therefore, we got a bit creative and used an arrange component with just text and needed styling for it to match, since is 2nd in the list, we followed the above references.
<img width="567" alt="screen shot 2018-10-31 at 3 57 02 pm" src="https://user-images.githubusercontent.com/1711642/47819254-e7f73100-dd27-11e8-8e29-2c1ed4349cf4.png">
